src/vocation.h

```diff
@@ -116,6 +120,8 @@ class Vocation
 
 		uint8_t soulMax = 100;
 		uint8_t clientId = 0;
+
+		bool _protected = false;
```

Comment:
💩... but `protected` is reserved within that scope 🤷

Reply:
`isProtected`?